From f2020ee4ee4b2f245637c77e81d62ab6d1db3c6c Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Tue, 25 Jan 2011 15:52:31 -0500
Subject: [PATCH] ignore RAISE_SETFCAP install failures

While the new RAISE_SETFCAP feature is nifty, its failure to run (often
due to the fs not supporting it) shouldn't impair the default install.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 progs/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/progs/Makefile b/progs/Makefile
index c261f57..9e1a84d 100644
--- a/progs/Makefile
+++ b/progs/Makefile
@@ -30,7 +30,7 @@ install: all
 		install -m 0755 $$p $(SBINDIR) ; \
 	done
 ifeq ($(RAISE_SETFCAP),yes)
-	$(SBINDIR)/setcap cap_setfcap=i $(SBINDIR)/setcap
+	-$(SBINDIR)/setcap cap_setfcap=i $(SBINDIR)/setcap
 endif
 
 clean:
-- 
1.7.4.rc2

